packages/block-library/src/term-template/index.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * Server-side rendering of the `core/term-template` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/term-template` block on the server.
+ *
+ * @since 6.x.x
```

Comment:
```suggestion
 * @since 6.9.0
```

And there's multiple other places to address it too.

Reply:
I wasn't sure whether to commit to 6.9.0, but we can always change it if it doesn't make it into the release. I've updated all these references in https://github.com/WordPress/gutenberg/pull/70720/commits/9294f16ee504d6b10a0402b233b4512da4745fbf.